test_scripts/UpdateVideoStreamingCapabilities/003_3_1_GetSystemCapability_after_OnSystemCapabilityUpdated_other_app.lua

```diff
@@ -0,0 +1,77 @@
+-- SDL does:
+-- - a. resend OnSystemCapabilityUpdated notification with updates of VIDEO_STREAMING capability type
+--    to the Application 1
+-- 2. Application 2 is registered and requests videoStreamingCapabilities via GetSystemCapability RPC
+-- SDL does:
+-- - a. send response to the Aplication 2 with received from HMI on startup videoStreamingCapabilities
+--    which stored internally
+-- - b. not request videoStreamingCapabilities from HMI
+---------------------------------------------------------------------------------------------------
+-- [[ Required Shared libraries ]]
+local common = require('test_scripts/UpdateVideoStreamingCapabilities/common')
+
+--[[ Local Variables ]]
+local appSessionId1 = 1
+local appSessionId2 = 2
+local expected = 1
+
+local vsc = common.getVideoStreamingCapability(5)
+vsc.additionalVideoStreamingCapabilities[1].preferredResolution = { resolutionWidth = 1920, resolutionHeight = 1080 }
+vsc.additionalVideoStreamingCapabilities[2].preferredResolution = { resolutionWidth = 1024, resolutionHeight = 768 }
+vsc.additionalVideoStreamingCapabilities[5].preferredResolution = { resolutionWidth = 15, resolutionHeight = 2 }
+
+local function getSystemCapability(pAppId)
+  local requestParams = {
+    systemCapabilityType = "VIDEO_STREAMING"
+  }
+  local responseParams = {
+    success = true,
+    resultCode = "SUCCESS",
+    systemCapability = {
+      systemCapabilityType = "VIDEO_STREAMING",
+      videoStreamingCapability = common.getVideoStreamingCapability()
+    }
+  }
+  local corId = common.getMobileSession(pAppId):SendRPC("GetSystemCapability", requestParams)
+  common.getHMIConnection():ExpectNotification("UI.GetCapabilities"):Times(0)
+  common.getMobileSession(pAppId):ExpectResponse(corId, responseParams)
+  :ValidIf(function(_, data)
+      if not common.isTableEqual(responseParams, data.payload) then
+        return false, "Parameters of the notification are incorrect: \nExpected: " .. common.toString(responseParams)
+          .. "\nActual: " .. common.toString(data.payload)
+      end
+      return true
+    end)
+  end
+
+--[[ Scenario ]]
+common.Title("Preconditions")
+common.Step("Clean environment", common.preconditions)
+common.Step("Set HMI Capabilities", common.setHMICapabilities)
+common.Step("Start SDL, HMI, connect Mobile, start Session", common.start, { common.hmiDefaultCapabilities })
+common.Step("Register App 1", common.registerAppWOPTU)
+common.Step("Activate App 1", common.activateApp)
+common.Step("GetSystemCapability with subscribe = true", common.getSystemCapability, { true })
+
+common.Title("Test")
+common.Step("OnSystemCapabilityUpdated", common.sendOnSystemCapabilityUpdated, { appSessionId1, expected, vsc })
+common.Step("GetSystemCapability", getSystemCapability, { appSessionId1 })
+common.Step("Register App 2", common.registerAppWOPTU, { appSessionId2 })
+common.Step("Activate App 2", common.activateApp, { appSessionId2 })
```

Comment:
```suggestion
```
By default GetSystemCapability is allowed in NONE HMI level, so the step with an app activation is redundant.